fedex_rest_api.gemspec

```diff
@@ -36,4 +36,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.17"
```

Comment:
I think we can remove the bundler dependency here, v1.17 is pretty dated and I don't think we wanna force applications into a certain version anyways